Session.js

```diff
@@ -676,17 +676,32 @@ Session.prototype = {
 	 * The value returned by the remote code. Only values that can be serialised to JSON, plus DOM elements, can be
 	 * returned.
 	 */
+
```

Comment:
Shouldn't be a blank line between a doc comment and the function.

Reply:
Didn't know that. Fixed.